### PR TITLE
api: refine pilot/frontgate/drone service spec

### PR DESCRIPTION
**Pull request by chai2010**

### CODE REVIEW

File: api/pilot.proto [hunk 1]
```diff
@@ -49,5 +50,95 @@ service PilotService {
...
+message ConfdInfo {
+	ConfdConfig config = 1;
+	ConfdBackendConfig backend_config = 2;
+}
+
+// See https://godoc.org/openpitrix.io/libconfd#Config
+message ConfdConfig {
+	google.protobuf.StringValue conf_dir = 1;
+	google.protobuf.Int32Value interval = 2;
+	google.protobuf.BoolValue noop =3;
```
Comment (chilianyi) [inline api/pilot.proto]: need format

File: api/frontgate/frontgate.proto [hunk 2]
```diff
@@ -19,15 +24,43 @@ service FrontgateService {
...
 	rpc DeregisterCmd (DeregisterCmdRequest) returns (Empty);
 
+	rpc GetSubTaskResult (GetSubTaskResultRequest) returns (SubTaskResult);
 	rpc ReportSubTaskResult (SubTaskResult) returns (Empty);
 }
 
+message Info {
+	string frontgate_id = 1;
+	string frontgate_node_id = 2;
+	string frontgate_node_ip = 3;
```
Comment (chilianyi) [inline api/frontgate/frontgate.proto]: repeated?

Comment (chai2010) [reply api/frontgate/frontgate.proto]: it is the current frontgate node info

File: api/pilot.proto (hunk 1 above)
Comment (chai2010) [reply api/pilot.proto]: done

**Pull request merged**